spec/features/apply_for_faculty_access_spec.rb

```diff
@@ -7,36 +7,7 @@
     @user.state = 'activated'
     @user.save!
   end
-
-  scenario "anonymous user asked to log in" do
-    visit faculty_access_apply_path(r: capybara_url(external_app_for_specs_path))
-
-    screenshot!
-
-    expect_sign_in_page
-    expect(page).to have_no_content("Sign up")
-
-    complete_login_username_or_email_screen('user')
-    complete_login_password_screen('password')
-
-    complete_faculty_access_apply_screen(
-      role: :other,
-      first_name: "Jimmy",
-      last_name: "Tudeski",
-      email: "howdy@ho.com",
-      phone_number: "000-0000",
-      school: "Rice University",
-      url: "http://www.rice.edu",
-      subjects: ["Biology", "Principles of Macroeconomics"],
-      newsletter: true,
-    )
-
-    expect(page).to have_content(t :"faculty_access.pending.page_heading")
-    click_button(t :"faculty_access.pending.ok")
-
-    expect(page.current_url).to eq(capybara_url(external_app_for_specs_path))
-  end
```

Comment:
I think we should leave this test in, just change the role that they pick to anything else. The other tests don't seem to test the flow where they visit the faculty then click to login (they login before going to the faculty page).

Reply:
@Dantemss - when I add the test back and change the role, it fails the test. Any ideas? I did try this before you mentioned it, I felt it wouldn't be favorable to remove tests. :)

Reply:
@Dantemss - Do you have a recommendation on how to fix the test besides changing the user role?